Assets/VRTK/Scripts/VRTK_AdaptiveQuality.cs

```diff
@@ -0,0 +1,708 @@
+// Adaptive Quality|Scripts|0240
+
+// Adapted from The Lab Renderer's ValveCamera.cs, available at
+// https://github.com/ValveSoftware/the_lab_renderer/blob/ae64c48a8ccbe5406aba1e39b160d4f2f7156c2c/Assets/TheLabRenderer/Scripts/ValveCamera.cs
+// For The Lab Renderer's license see THIRD_PARTY_NOTICES.
+namespace VRTK
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Text;
+    using UnityEngine;
+    using UnityEngine.VR;
+
+    /// <summary>
+    /// Adaptive Quality dynamically changes rendering settings to maintain VR framerate while maximizing GPU utilization.
+    /// </summary>
+    /// <remarks>
+    /// The Adaptive Quality script is attached to the `eye` object within the `[CameraRig]` prefab.
+    /// <para>&#160;</para>
+    /// There are two goals:
+    /// <list type="bullet">
+    /// <item> <description>Reduce the chances of dropping frames and reprojecting</description> </item>
+    /// <item> <description>Increase quality when there are idle GPU cycles</description> </item>
+    /// </list>
+    /// <para />
+    /// This script currently changes the following to reach these goals:
+    /// <list type="bullet">
+    /// <item> <description>Rendering resolution and viewport (aka Dynamic Resolution)</description> </item>
+    /// </list>
+    /// <para />
+    /// In the future it could be changed to also change the following:
+    /// <list type="bullet">
+    /// <item> <description>MSAA level</description> </item>
+    /// <item> <description>Fixed Foveated Rendering</description> </item>
+    /// <item> <description>Radial Density Masking</description> </item>
+    /// <item> <description>(Non-fixed) Foveated Rendering (once HMDs support eye tracking)</description> </item>
+    /// <item> <description>...</description> </item>
```

Comment:
remove this line maybe?

it doesn't really mean anything to have `...` at the end of the list does it?